tests/test.vader

```diff
@@ -1366,59 +1369,76 @@ Expect:
   2ghi 2jkl
   
 
+" Here it deletes as linewise because of :h d-special
+" (Help tag only exists in Vim)
```

Comment:
Should these comments use `#` prefix?

Reply:
Oop, yes

[ada981b](https://github.com/justinmk/vim-sneak/pull/326/commits/ada981b1c65dedefd7693fb2caecc5558b497475)